CMakeLists.txt

```
@@ -399,7 +399,9 @@ add_subdirectory (io)
 add_subdirectory (net)
 add_subdirectory (graf2d)
 add_subdirectory (graf3d)
-add_subdirectory (gui)
+if(TARGET Gui)
```

```suggestion
if(NOT minimal)
```
The target Gui is only defined in gui/CMakeLists.txt
`ROOT_STANDARD_LIBRARY_PACKAGE(Gui`

so the target is never defined because the subdirectory is not added?